generate_config.sh

```diff
@@ -117,6 +117,9 @@ cat << EOF > mailcow.conf
 
 MAILCOW_HOSTNAME=${MAILCOW_HOSTNAME}
 
+# Password hash algorithm
```

Comment:
We could add a link to a new docs page to inform about supported methods, what do you think?